### ISSUE TITLE
Research paper

### ISSUE THREAD

**Issue by JinraeKim** [peter-nebe/stair-step-detector#1]
Is there any related research papers that used this project?

**Comment by peter-nebe** [comment]
Yes of course. This was a private "research project" that I did alongside my main job. The goal was to show what modern sensors can do and to enable the practical application of this innovation in the field of building measurement or the control of mobile robots. Revolutionary hardware needs revolutionary software for groundbreaking applications.

If you are an application developer and interested in the software module, I will try to support you in integrating the module into your system.

I am also happy about scientists and researchers who have questions or ideas for further development.
